jboss/container/wildfly/launch/json-logging/added/launch/json_logging.sh

```diff
@@ -26,27 +26,26 @@ function configureByMarkers() {
 
 function configureByCLI() {
   if [ "${ENABLE_JSON_LOGGING^^}" == "TRUE" ]; then
-    cat <<'EOF' >> ${CLI_SCRIPT_FILE}
-      if (outcome != success) of /subsystem=logging/json-formatter=OPENSHIFT:read-resource
-        /subsystem=logging/json-formatter=OPENSHIFT:add(exception-output-type=formatted, key-overrides=[timestamp="@timestamp"], meta-data=[@version=1])
-      else
-        /subsystem=logging/json-formatter=OPENSHIFT:write-attribute(name=exception-output-type, value=formatted)
-        /subsystem=logging/json-formatter=OPENSHIFT:write-attribute(name=key-overrides, value=[timestamp="@timestamp"]
-        /subsystem=logging/json-formatter=OPENSHIFT:write-attribute(name=meta-data, value=[@version=1])
-      end-if
+
+    # We cannot have nested if sentences in CLI, so we use Xpath here to see if the subsystem=logging is in the file
+    xpath="\"//*[local-name()='subsystem' and starts-with(namespace-uri(), 'urn:jboss:domain:logging:')]\""
+    local ret
+    testXpathExpression "${xpath}" "ret"
+
+    if [ "${ret}" -eq 0 ]; then
+      cat <<'EOF' >> ${CLI_SCRIPT_FILE}
+          if (outcome != success) of /subsystem=logging/json-formatter=OPENSHIFT:read-resource
+            /subsystem=logging/json-formatter=OPENSHIFT:add(exception-output-type=formatted, key-overrides=[timestamp="@timestamp"], meta-data=[@version=1])
+          else
+            /subsystem=logging/json-formatter=OPENSHIFT:write-attribute(name=exception-output-type, value=formatted)
+            /subsystem=logging/json-formatter=OPENSHIFT:write-attribute(name=key-overrides, value=[timestamp="@timestamp"]
+            /subsystem=logging/json-formatter=OPENSHIFT:write-attribute(name=meta-data, value=[@version=1])
+          end-if
 EOF
-    consoleHandlerName "OPENSHIFT" >> ${CLI_SCRIPT_FILE}
+      consoleHandlerName "OPENSHIFT" >> ${CLI_SCRIPT_FILE}
+    fi
     sed -i 's|##CONSOLE-FORMATTER##|OPENSHIFT|' $LOGGING_FILE
   else
```

Comment:
Now I think we can even go further and in case ENABLE_JSON_LOGGING is not set or it is false, remove, if exists, the console handler configuration in case it is using OPENSHIFT

Reply:
Why would we remove it? You mean in the management model?

Reply:
Yes, in the management model, when we are using CLI, at least if ENABLE_JSON_LOGGING is false; even if I have a server config file that it is using OPENSHIFT by default as named-formatter for CONSOLE handler, I would expect to do not use it, because I am explicitly setting the env var to false

Reply:
@kabir We agreed this PR can be in as it is for now. We still have to discuss better the meaning of ENABLE_JSON_LOGGING environment variable, but it would be nice to get this merged to Jean Francois can test it with wildfly-s2i